runtime/compiler/control/J9Options.cpp

```diff
@@ -1599,6 +1604,31 @@ void J9::Options::preProcessJniAccelerator(J9JavaVM *vm)
       }
    }
 
+uint64_t getCodeCacheMaxPercentageOfAvailableMemory(J9JavaVM *vm)
```

Comment:
The value that is returned by this method is a `double`, so it's better to change the prototype of this method to reflect that